ci-operator/jobs/openshift/origin/openshift-origin-release-3.11-periodics.yaml

```diff
@@ -3,33 +3,30 @@ periodics:
   cluster: build02
   cron: 20 4 * * *
   decorate: true
-  decoration_config:
-    skip_cloning: true
```

Comment:
Likely this is what broke it - periodic should still have it cloned so it can read the file? @petr-muller

Reply:
Yeah, that's why it was [broken](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-origin-release-3.11-e2e-gcp/1396682492311769088) *before*, the job was not generated so it wasn't cloning (note that removing this stanza makes it clone = fixes it). The rehearsal fails on:

```
ERRO[2021-05-24T14:53:37Z] Some steps failed:                           
ERRO[2021-05-24T14:53:37Z]   * could not run steps: step e2e failed: template e2e has required parameter RPM_REPO_OPENSHIFT_ORIGIN which is not defined 
```

Which the handcrafted job had before, generator removed it ([intentionally](https://github.com/openshift/ci-tools/blob/4cf25561835d4fa532811b8ca3d51dc5ab56ea54/pkg/prowgen/prowgen.go#L409-L419)). The identical *presubmit* job (generated from same test stanza, just without `cron`) [worked](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/25875/pull-ci-openshift-origin-release-3.11-e2e-gcp/1390678228946915328) as of May 7.

So either we broke these params since then, or something is broken in ci-op. But I don't care about 3.11 that much to investigate right now.

Reply:
yuck...

Reply:
:vomiting_face: indeed